pallets/messages/src/weights.rs

```diff
@@ -66,34 +67,34 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: Msa KeyInfoOf (r:1 w:0)
 	// Storage: Messages BlockMessages (r:1 w:1)
 	fn add_onchain_message(n: u32, m: u32, ) -> Weight {
-		(33_905_000 as Weight)
+		(0 as Weight)
```

Comment:
@shannonwells, This is the only one that has zeros here. I haven't looked deep enough into the weights to understand why. Do you know? It feels wrong or perhaps our benchmark setup for the messages pallet is wrong?

Reply:
Thanks for the reminder; I meant to check that out, to see if there had been major changes to messages since the weights were last regenerated.

Reply:
I talked to @aramikm - it's actually fine, it means that the benchmark run found a good linear relationship when running all the benchmark instances